test/tests/getClassForDocument.test.ts

```diff
@@ -45,12 +46,18 @@ export function suite() {
     expect(userReflectedType).to.not.equals(CarType);
   });
 
+  it(`should return correct class type for document also using typeAlias`, async () => {
+    const event = new EventModel();
```

Comment:
but are you *sure* that it is just this model, when only having one named "Event"?
-> please add to be *sure* and dummy Event and make it different properties, thanks

Reply:
Yes, I'm sure. As you asked, I've added another model named Event with different properties and typeAlias.